homeworks/rostyslav.havuka_RostyslavHavuka/11-dependency-injection/vehicle/src/app/app.module.ts

```diff
@@ -0,0 +1,36 @@
+import { BrowserModule } from '@angular/platform-browser';
+import { NgModule } from '@angular/core';
+import { AppRoutingModule } from './app-routing.module';
+import { AppComponent } from './app.component';
+import { appServices } from './services';
+import { InfoCarPipe } from './pipes/info-car.pipe';
```

Comment:
I showed an example of how you can import your pipes in the lecture.

Create the `appPipes` variable and use it here. Remove other imports.
Same for directives 😄